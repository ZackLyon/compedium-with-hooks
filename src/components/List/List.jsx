import React from 'react';
import Pokemon from '../Pokemon/Pokemon';
import './List.css';

export default function List({ pokedex }) {
  return (
    <div className='list-layout'>
      {pokedex.map((pokemon) => (
        <Pokemon poke={pokemon} key={pokemon.pokemon} />
      ))}
    </div>
  );
}
