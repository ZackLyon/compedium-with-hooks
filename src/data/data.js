const data = [
  {
    _id: '5ff4fb7cd89993a89cc6563f',
    pokemon: 'abomasnow',
    id: 512,
    species_id: 460,
    height: 22,
    weight: 1355,
    base_experience: 173,
    type_1: 'grass',
    type_2: 'ice',
    attack: 92,
    defense: 75,
    hp: 90,
    special_attack: 92,
    special_defense: 85,
    speed: 60,
    ability_1: 'snow-warning',
    ability_2: 'NA',
    ability_hidden: 'soundproof',
    color_1: '#78C850',
    color_2: '#98D8D8',
    color_f: '#7FCB70',
    egg_group_1: 'monster',
    egg_group_2: 'plant',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/460.png',
    generation_id: 4,
    evolves_from_species_id: '459',
    evolution_chain_id: 239,
    shape_id: 6,
    shape: 'upright',
    pokebase: 'abomasnow',
    pokedex: 'http://www.pokemon.com/us/pokedex/abomasnow',
  },
  {
    _id: '5ff4fb7cd89993a89cc65641',
    pokemon: 'abomasnow-mega',
    id: 513,
    species_id: 460,
    height: 27,
    weight: 1850,
    base_experience: 208,
    type_1: 'grass',
    type_2: 'ice',
    attack: 132,
    defense: 105,
    hp: 90,
    special_attack: 132,
    special_defense: 105,
    speed: 30,
    ability_1: 'snow-warning',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#78C850',
    color_2: '#98D8D8',
    color_f: '#7FCB70',
    egg_group_1: 'monster',
    egg_group_2: 'plant',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/460_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'abomasnow',
    pokedex: 'http://www.pokemon.com/us/pokedex/abomasnow',
  },
  {
    _id: '5ff4fb7cd89993a89cc65488',
    pokemon: 'abra',
    id: 69,
    species_id: 63,
    height: 9,
    weight: 195,
    base_experience: 62,
    type_1: 'psychic',
    type_2: 'NA',
    attack: 20,
    defense: 15,
    hp: 25,
    special_attack: 105,
    special_defense: 55,
    speed: 90,
    ability_1: 'synchronize',
    ability_2: 'inner-focus',
    ability_hidden: 'magic-guard',
    color_1: '#F85888',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'humanshape',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/063.png',
    generation_id: 1,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 27,
    shape_id: 6,
    shape: 'upright',
    pokebase: 'abra',
    pokedex: 'http://www.pokemon.com/us/pokedex/abra',
  },
  {
    _id: '5ff4fb7cd89993a89cc655cb',
    pokemon: 'absol',
    id: 396,
    species_id: 359,
    height: 12,
    weight: 470,
    base_experience: 163,
    type_1: 'dark',
    type_2: 'NA',
    attack: 130,
    defense: 60,
    hp: 65,
    special_attack: 75,
    special_defense: 60,
    speed: 75,
    ability_1: 'pressure',
    ability_2: 'super-luck',
    ability_hidden: 'justified',
    color_1: '#705848',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'ground',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/359.png',
    generation_id: 3,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 185,
    shape_id: 8,
    shape: 'quadruped',
    pokebase: 'absol',
    pokedex: 'http://www.pokemon.com/us/pokedex/absol',
  },
  {
    _id: '5ff4fb7cd89993a89cc655ce',
    pokemon: 'absol-mega',
    id: 397,
    species_id: 359,
    height: 12,
    weight: 490,
    base_experience: 198,
    type_1: 'dark',
    type_2: 'NA',
    attack: 150,
    defense: 60,
    hp: 65,
    special_attack: 115,
    special_defense: 60,
    speed: 115,
    ability_1: 'magic-bounce',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#705848',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'ground',
    egg_group_2: 'NA',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/359_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'absol',
    pokedex: 'http://www.pokemon.com/us/pokedex/absol',
  },
  {
    _id: '5ff4fb7cd89993a89cc656ea',
    pokemon: 'accelgor',
    id: 681,
    species_id: 617,
    height: 8,
    weight: 253,
    base_experience: 173,
    type_1: 'bug',
    type_2: 'NA',
    attack: 70,
    defense: 40,
    hp: 80,
    special_attack: 100,
    special_defense: 60,
    speed: 145,
    ability_1: 'hydration',
    ability_2: 'sticky-hold',
    ability_hidden: 'unburden',
    color_1: '#A8B820',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'bug',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/617.png',
    generation_id: 5,
    evolves_from_species_id: '616',
    evolution_chain_id: 312,
    shape_id: 4,
    shape: 'arms',
    pokebase: 'accelgor',
    pokedex: 'http://www.pokemon.com/us/pokedex/accelgor',
  },
  {
    _id: '5ff4fb7cd89993a89cc65733',
    pokemon: 'aegislash-blade',
    id: 753,
    species_id: 681,
    height: 17,
    weight: 530,
    base_experience: 234,
    type_1: 'steel',
    type_2: 'ghost',
    attack: 150,
    defense: 50,
    hp: 60,
    special_attack: 150,
    special_defense: 50,
    speed: 60,
    ability_1: 'stance-change',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#B8B8D0',
    color_2: '#705898',
    color_f: '#A6A0C2',
    egg_group_1: 'mineral',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/681.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'aegislash',
    pokedex: 'http://www.pokemon.com/us/pokedex/aegislash',
  },
  {
    _id: '5ff4fb7cd89993a89cc65745',
    pokemon: 'aegislash-shield',
    id: 754,
    species_id: 681,
    height: 17,
    weight: 530,
    base_experience: 234,
    type_1: 'steel',
    type_2: 'ghost',
    attack: 50,
    defense: 150,
    hp: 60,
    special_attack: 50,
    special_defense: 150,
    speed: 60,
    ability_1: 'stance-change',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#B8B8D0',
    color_2: '#705898',
    color_f: '#A6A0C2',
    egg_group_1: 'mineral',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/681.png',
    generation_id: 6,
    evolves_from_species_id: '680',
    evolution_chain_id: 349,
    shape_id: 5,
    shape: 'blob',
    pokebase: 'aegislash',
    pokedex: 'http://www.pokemon.com/us/pokedex/aegislash',
  },
  {
    _id: '5ff4fb7cd89993a89cc654d7',
    pokemon: 'aerodactyl',
    id: 154,
    species_id: 142,
    height: 18,
    weight: 590,
    base_experience: 180,
    type_1: 'rock',
    type_2: 'flying',
    attack: 105,
    defense: 65,
    hp: 80,
    special_attack: 60,
    special_defense: 75,
    speed: 130,
    ability_1: 'rock-head',
    ability_2: 'pressure',
    ability_hidden: 'unnerve',
    color_1: '#B8A038',
    color_2: '#A890F0',
    color_f: '#B49C64',
    egg_group_1: 'flying',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/142.png',
    generation_id: 1,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 71,
    shape_id: 9,
    shape: 'wings',
    pokebase: 'aerodactyl',
    pokedex: 'http://www.pokemon.com/us/pokedex/aerodactyl',
  },
  {
    _id: '5ff4fb7cd89993a89cc654d8',
    pokemon: 'aerodactyl-mega',
    id: 155,
    species_id: 142,
    height: 21,
    weight: 790,
    base_experience: 215,
    type_1: 'rock',
    type_2: 'flying',
    attack: 135,
    defense: 85,
    hp: 80,
    special_attack: 70,
    special_defense: 95,
    speed: 150,
    ability_1: 'tough-claws',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#B8A038',
    color_2: '#A890F0',
    color_f: '#B49C64',
    egg_group_1: 'flying',
    egg_group_2: 'NA',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/142_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'aerodactyl',
    pokedex: 'http://www.pokemon.com/us/pokedex/aerodactyl',
  },
  {
    _id: '5ff4fb7cd89993a89cc65592',
    pokemon: 'aggron',
    id: 333,
    species_id: 306,
    height: 21,
    weight: 3600,
    base_experience: 239,
    type_1: 'steel',
    type_2: 'rock',
    attack: 110,
    defense: 180,
    hp: 70,
    special_attack: 60,
    special_defense: 60,
    speed: 50,
    ability_1: 'sturdy',
    ability_2: 'rock-head',
    ability_hidden: 'heavy-metal',
    color_1: '#B8B8D0',
    color_2: '#B8A038',
    color_f: '#B8B2AB',
    egg_group_1: 'monster',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/306.png',
    generation_id: 3,
    evolves_from_species_id: '305',
    evolution_chain_id: 151,
    shape_id: 6,
    shape: 'upright',
    pokebase: 'aggron',
    pokedex: 'http://www.pokemon.com/us/pokedex/aggron',
  },
  {
    _id: '5ff4fb7cd89993a89cc6558a',
    pokemon: 'aggron-mega',
    id: 334,
    species_id: 306,
    height: 22,
    weight: 3950,
    base_experience: 284,
    type_1: 'steel',
    type_2: 'NA',
    attack: 140,
    defense: 230,
    hp: 70,
    special_attack: 60,
    special_defense: 80,
    speed: 50,
    ability_1: 'filter',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#B8B8D0',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'monster',
    egg_group_2: 'NA',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/306_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'aggron',
    pokedex: 'http://www.pokemon.com/us/pokedex/aggron',
  },
  {
    _id: '5ff4fb7cd89993a89cc65510',
    pokemon: 'aipom',
    id: 206,
    species_id: 190,
    height: 8,
    weight: 115,
    base_experience: 72,
    type_1: 'normal',
    type_2: 'NA',
    attack: 70,
    defense: 55,
    hp: 55,
    special_attack: 40,
    special_defense: 55,
    speed: 85,
    ability_1: 'run-away',
    ability_2: 'pickup',
    ability_hidden: 'skill-link',
    color_1: '#A8A878',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'ground',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/190.png',
    generation_id: 2,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 93,
    shape_id: 6,
    shape: 'upright',
    pokebase: 'aipom',
    pokedex: 'http://www.pokemon.com/us/pokedex/aipom',
  },
  {
    _id: '5ff4fb7cd89993a89cc6548d',
    pokemon: 'alakazam',
    id: 71,
    species_id: 65,
    height: 15,
    weight: 480,
    base_experience: 225,
    type_1: 'psychic',
    type_2: 'NA',
    attack: 50,
    defense: 45,
    hp: 55,
    special_attack: 135,
    special_defense: 95,
    speed: 120,
    ability_1: 'synchronize',
    ability_2: 'inner-focus',
    ability_hidden: 'magic-guard',
    color_1: '#F85888',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'humanshape',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/065.png',
    generation_id: 1,
    evolves_from_species_id: '64',
    evolution_chain_id: 27,
    shape_id: 12,
    shape: 'humanoid',
    pokebase: 'alakazam',
    pokedex: 'http://www.pokemon.com/us/pokedex/alakazam',
  },
  {
    _id: '5ff4fb7cd89993a89cc6548c',
    pokemon: 'alakazam-mega',
    id: 72,
    species_id: 65,
    height: 12,
    weight: 480,
    base_experience: 266,
    type_1: 'psychic',
    type_2: 'NA',
    attack: 50,
    defense: 65,
    hp: 55,
    special_attack: 175,
    special_defense: 95,
    speed: 150,
    ability_1: 'trace',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#F85888',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'humanshape',
    egg_group_2: 'NA',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/065_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'alakazam',
    pokedex: 'http://www.pokemon.com/us/pokedex/alakazam',
  },
  {
    _id: '5ff4fb7cd89993a89cc656d5',
    pokemon: 'alomomola',
    id: 658,
    species_id: 594,
    height: 12,
    weight: 316,
    base_experience: 165,
    type_1: 'water',
    type_2: 'NA',
    attack: 75,
    defense: 80,
    hp: 165,
    special_attack: 40,
    special_defense: 45,
    speed: 65,
    ability_1: 'healer',
    ability_2: 'hydration',
    ability_hidden: 'regenerator',
    color_1: '#6890F0',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'water1',
    egg_group_2: 'water2',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/594.png',
    generation_id: 5,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 302,
    shape_id: 3,
    shape: 'fish',
    pokebase: 'alomomola',
    pokedex: 'http://www.pokemon.com/us/pokedex/alomomola',
  },
  {
    _id: '5ff4fb7cd89993a89cc655b0',
    pokemon: 'altaria',
    id: 366,
    species_id: 334,
    height: 11,
    weight: 206,
    base_experience: 172,
    type_1: 'dragon',
    type_2: 'flying',
    attack: 70,
    defense: 90,
    hp: 75,
    special_attack: 70,
    special_defense: 105,
    speed: 80,
    ability_1: 'natural-cure',
    ability_2: 'NA',
    ability_hidden: 'cloud-nine',
    color_1: '#7038F8',
    color_2: '#A890F0',
    color_f: '#7D4DF6',
    egg_group_1: 'flying',
    egg_group_2: 'dragon',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/334.png',
    generation_id: 3,
    evolves_from_species_id: '333',
    evolution_chain_id: 168,
    shape_id: 9,
    shape: 'wings',
    pokebase: 'altaria',
    pokedex: 'http://www.pokemon.com/us/pokedex/altaria',
  },
  {
    _id: '5ff4fb7cd89993a89cc655b1',
    pokemon: 'altaria-mega',
    id: 367,
    species_id: 334,
    height: 150,
    weight: 206,
    base_experience: 207,
    type_1: 'dragon',
    type_2: 'fairy',
    attack: 110,
    defense: 110,
    hp: 75,
    special_attack: 110,
    special_defense: 105,
    speed: 80,
    ability_1: 'pixilate',
    ability_2: 'NA',
    ability_hidden: 'NA',
    color_1: '#7038F8',
    color_2: '#EE99AC',
    color_f: '#8E4FE5',
    egg_group_1: 'flying',
    egg_group_2: 'dragon',
    url_image:
      'http://assets.pokemon.com/assets/cms2/img/pokedex/full/334_f2.png',
    generation_id: 'NA',
    evolves_from_species_id: 'NA',
    evolution_chain_id: 'NA',
    shape_id: 'NA',
    shape: 'NA',
    pokebase: 'altaria',
    pokedex: 'http://www.pokemon.com/us/pokedex/altaria',
  },
  {
    _id: '5ff4fb7cd89993a89cc65741',
    pokemon: 'amaura',
    id: 771,
    species_id: 698,
    height: 13,
    weight: 252,
    base_experience: 72,
    type_1: 'rock',
    type_2: 'ice',
    attack: 59,
    defense: 50,
    hp: 77,
    special_attack: 67,
    special_defense: 63,
    speed: 46,
    ability_1: 'refrigerate',
    ability_2: 'NA',
    ability_hidden: 'snow-warning',
    color_1: '#B8A038',
    color_2: '#98D8D8',
    color_f: '#B0AD5E',
    egg_group_1: 'monster',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/698.png',
    generation_id: 6,
    evolves_from_species_id: 'NA',
    evolution_chain_id: 358,
    shape_id: 8,
    shape: 'quadruped',
    pokebase: 'amaura',
    pokedex: 'http://www.pokemon.com/us/pokedex/amaura',
  },
  {
    _id: '5ff4fb7cd89993a89cc65615',
    pokemon: 'ambipom',
    id: 473,
    species_id: 424,
    height: 12,
    weight: 203,
    base_experience: 169,
    type_1: 'normal',
    type_2: 'NA',
    attack: 100,
    defense: 66,
    hp: 75,
    special_attack: 60,
    special_defense: 66,
    speed: 115,
    ability_1: 'technician',
    ability_2: 'pickup',
    ability_hidden: 'skill-link',
    color_1: '#A8A878',
    color_2: 'NA',
    color_f: 'NA',
    egg_group_1: 'ground',
    egg_group_2: 'NA',
    url_image: 'http://assets.pokemon.com/assets/cms2/img/pokedex/full/424.png',
    generation_id: 4,
    evolves_from_species_id: '190',
    evolution_chain_id: 93,
    shape_id: 6,
    shape: 'upright',
    pokebase: 'ambipom',
    pokedex: 'http://www.pokemon.com/us/pokedex/ambipom',
  },
];

export { data };
