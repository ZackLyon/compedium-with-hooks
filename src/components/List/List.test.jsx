import List from './List.jsx';
import { render } from '@testing-library/react';
import { data } from '../../data/data.js';

it('should match the screenshot of the displayed list of pokemon', () => {
  const { container } = render(<List pokedex={data} />);

  expect(container).toMatchSnapshot();
});
