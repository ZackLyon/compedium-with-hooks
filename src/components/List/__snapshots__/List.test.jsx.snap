// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the screenshot of the displayed list of pokemon 1`] = `
<div>
  <div
    class="list-layout"
  >
    <div
      class="pokemon-card"
    >
      <img
        alt="abomasnow"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/460.png"
      />
      <h1
        class="name"
      >
        abomasnow
      </h1>
      <section
        class="stats"
      >
        <div>
          grass
          /ice
        </div>
        <div>
          Attack: 
          92
        </div>
        <div>
          Defense: 
          75
        </div>
        <div>
          HP: 
          90
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="abomasnow-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/460_f2.png"
      />
      <h1
        class="name"
      >
        abomasnow-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          grass
          /ice
        </div>
        <div>
          Attack: 
          132
        </div>
        <div>
          Defense: 
          105
        </div>
        <div>
          HP: 
          90
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="abra"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/063.png"
      />
      <h1
        class="name"
      >
        abra
      </h1>
      <section
        class="stats"
      >
        <div>
          psychic
          
        </div>
        <div>
          Attack: 
          20
        </div>
        <div>
          Defense: 
          15
        </div>
        <div>
          HP: 
          25
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="absol"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/359.png"
      />
      <h1
        class="name"
      >
        absol
      </h1>
      <section
        class="stats"
      >
        <div>
          dark
          
        </div>
        <div>
          Attack: 
          130
        </div>
        <div>
          Defense: 
          60
        </div>
        <div>
          HP: 
          65
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="absol-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/359_f2.png"
      />
      <h1
        class="name"
      >
        absol-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          dark
          
        </div>
        <div>
          Attack: 
          150
        </div>
        <div>
          Defense: 
          60
        </div>
        <div>
          HP: 
          65
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="accelgor"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/617.png"
      />
      <h1
        class="name"
      >
        accelgor
      </h1>
      <section
        class="stats"
      >
        <div>
          bug
          
        </div>
        <div>
          Attack: 
          70
        </div>
        <div>
          Defense: 
          40
        </div>
        <div>
          HP: 
          80
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aegislash-blade"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/681.png"
      />
      <h1
        class="name"
      >
        aegislash-blade
      </h1>
      <section
        class="stats"
      >
        <div>
          steel
          /ghost
        </div>
        <div>
          Attack: 
          150
        </div>
        <div>
          Defense: 
          50
        </div>
        <div>
          HP: 
          60
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aegislash-shield"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/681.png"
      />
      <h1
        class="name"
      >
        aegislash-shield
      </h1>
      <section
        class="stats"
      >
        <div>
          steel
          /ghost
        </div>
        <div>
          Attack: 
          50
        </div>
        <div>
          Defense: 
          150
        </div>
        <div>
          HP: 
          60
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aerodactyl"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/142.png"
      />
      <h1
        class="name"
      >
        aerodactyl
      </h1>
      <section
        class="stats"
      >
        <div>
          rock
          /flying
        </div>
        <div>
          Attack: 
          105
        </div>
        <div>
          Defense: 
          65
        </div>
        <div>
          HP: 
          80
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aerodactyl-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/142_f2.png"
      />
      <h1
        class="name"
      >
        aerodactyl-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          rock
          /flying
        </div>
        <div>
          Attack: 
          135
        </div>
        <div>
          Defense: 
          85
        </div>
        <div>
          HP: 
          80
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aggron"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/306.png"
      />
      <h1
        class="name"
      >
        aggron
      </h1>
      <section
        class="stats"
      >
        <div>
          steel
          /rock
        </div>
        <div>
          Attack: 
          110
        </div>
        <div>
          Defense: 
          180
        </div>
        <div>
          HP: 
          70
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aggron-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/306_f2.png"
      />
      <h1
        class="name"
      >
        aggron-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          steel
          
        </div>
        <div>
          Attack: 
          140
        </div>
        <div>
          Defense: 
          230
        </div>
        <div>
          HP: 
          70
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="aipom"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/190.png"
      />
      <h1
        class="name"
      >
        aipom
      </h1>
      <section
        class="stats"
      >
        <div>
          normal
          
        </div>
        <div>
          Attack: 
          70
        </div>
        <div>
          Defense: 
          55
        </div>
        <div>
          HP: 
          55
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="alakazam"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/065.png"
      />
      <h1
        class="name"
      >
        alakazam
      </h1>
      <section
        class="stats"
      >
        <div>
          psychic
          
        </div>
        <div>
          Attack: 
          50
        </div>
        <div>
          Defense: 
          45
        </div>
        <div>
          HP: 
          55
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="alakazam-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/065_f2.png"
      />
      <h1
        class="name"
      >
        alakazam-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          psychic
          
        </div>
        <div>
          Attack: 
          50
        </div>
        <div>
          Defense: 
          65
        </div>
        <div>
          HP: 
          55
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="alomomola"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/594.png"
      />
      <h1
        class="name"
      >
        alomomola
      </h1>
      <section
        class="stats"
      >
        <div>
          water
          
        </div>
        <div>
          Attack: 
          75
        </div>
        <div>
          Defense: 
          80
        </div>
        <div>
          HP: 
          165
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="altaria"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/334.png"
      />
      <h1
        class="name"
      >
        altaria
      </h1>
      <section
        class="stats"
      >
        <div>
          dragon
          /flying
        </div>
        <div>
          Attack: 
          70
        </div>
        <div>
          Defense: 
          90
        </div>
        <div>
          HP: 
          75
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="altaria-mega"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/334_f2.png"
      />
      <h1
        class="name"
      >
        altaria-mega
      </h1>
      <section
        class="stats"
      >
        <div>
          dragon
          /fairy
        </div>
        <div>
          Attack: 
          110
        </div>
        <div>
          Defense: 
          110
        </div>
        <div>
          HP: 
          75
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="amaura"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/698.png"
      />
      <h1
        class="name"
      >
        amaura
      </h1>
      <section
        class="stats"
      >
        <div>
          rock
          /ice
        </div>
        <div>
          Attack: 
          59
        </div>
        <div>
          Defense: 
          50
        </div>
        <div>
          HP: 
          77
        </div>
      </section>
    </div>
    <div
      class="pokemon-card"
    >
      <img
        alt="ambipom"
        src="http://assets.pokemon.com/assets/cms2/img/pokedex/full/424.png"
      />
      <h1
        class="name"
      >
        ambipom
      </h1>
      <section
        class="stats"
      >
        <div>
          normal
          
        </div>
        <div>
          Attack: 
          100
        </div>
        <div>
          Defense: 
          66
        </div>
        <div>
          HP: 
          75
        </div>
      </section>
    </div>
  </div>
</div>
`;
